sp800-63b/sec5_authenticators.md

```diff
@@ -359,7 +359,7 @@ Biometric matching SHOULD be performed locally on claimant's device or MAY be pe
 
 If matching is performed centrally:
 
-* Use of the biometric SHALL be bound tightly to a single, specific device that is identified using approved cryptography.
+* Use of the biometric SHALL be limited to a single, specific device that is identified using approved cryptography.
```

Comment:
This isn't what we mean.  If my fingerprint is on the server, and the agency wants to issue me a laptop, ipad, iphone, samsung, they should - each being trusted and revocable.